### DISCUSSION TITLE
Track enemy strategy point gain

### DISCUSSION THREAD

**Discussion by TrentBartlem** [BSData/adeptus-titanicus#89] [Ideas]
Every different legio selected in a battleforce (or household) adds +2 strategy points to the enemy's total.

Is this something we should track?

**Comment by ghost** [comment]
+1

**Reply by TrentBartlem** [reply]
so would it be another type of cost, like our own strategy points, except they go up?

**Reply by TrentBartlem** [reply]
would this mean it would show up on list creation as something you'd have to choose, though? like

points: 1500
strategem points: 5
**stratagem points given to enemy: -1**

?

**Reply by ghost** [reply]
It would be present there if it was a cost. 
I think we could add buttons or something in Uncategorized block if it is incremented during play? I haven't re-read that second and don't recall quite how it works.

**Reply by TrentBartlem** [reply]
it doesn't happen during play, it just gives extra points at the start of the battle for every legio on the enemy force (including support titans, so a Household with unaffiliated titans doesn't cede any strat points)
